app/liquid/views/layouts/member-registration.liquid

```diff
@@ -7,24 +7,30 @@
     <div class="imageless-title">
       <div class="typography__highlight-wrapper">
         <h1 class="typography__highlight">
-          <span>Member Registration</span>
+          <span>{{ 'fundraiser.thank_you' | t }}</span>
         </h1>
       </div>
     </div>
 
     <div class="body-text main-feature main-feature--close-top">
-        <p>
-        </p>
 
+      <h4>{{ 'member_registration.create_account' | t }}</h4>
+
+      <p>{{ 'member_registration.create_account_explanation' | t }} <br>
+        <span>{{ 'member_registration.form.your_email' | t }}: {{ email }}</span> <br>
```

Comment:
Not asking for an email, but rather displaying it (as per your mockups):

![image](https://cloud.githubusercontent.com/assets/12949/18636293/1d842f2a-7e80-11e6-996d-14497e9116b0.png)